src/core/Store.ts

```diff
@@ -25,26 +26,36 @@ class Store extends OrbitdbStore {
 
     this._index.store = this;
 
-    this.events.on("write", (address, entry) => {
-      this._index.handleEntry(entry);
-    });
-    this.events.on("replicate.progress", (address, hash, entry) => {
+    this.events.on("write", (address: any, entry: any) => {
```

Comment:
Why are `address` and `entry` of `any` type?

Reply:
I wasn't sure about these. I'll log and see what can the possible types. Maybe @vaultec81 can tell us.

Reply:
Address should be type of string. Entry should be type of object.